test/units/testsuite-53.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+# This test makes some basic checks that RPM macros work correctly.
+# RPM is a simple C program available on different Linux distros, not only RPM-based ones,
+# and even BSD systems, so it must not be a problem to require it.
+# rpmspec utility is required (so this test will work with RPM 4 but won't work with RPM 5).
+set -xefu
+
+trap 'echo "Some tests of RPM macros failed!"' ERR
+trap 'rm -f $TMP_RPM_SPEC $crp_tmp' EXIT
+
+RPM_MACROS_FILE="${TEST_BASE_DIR}/../src/core/macros.systemd.in"
+
+if ! command -v rpmspec >/dev/null 2>&1 ; then
+	echo "rpmspec(8) is needed to run test 52!"
+	exit 1
+fi
+
+if [ ! -f "$RPM_MACROS_FILE" ]; then
+	echo "RPM macros file not found in ${RPM_MACROS_FILE}!"
+	exit 1
+fi
+
+TMP_RPM_SPEC="$(mktemp)"
+RPM_SPEC_HEADER="
+Summary: systemd test 52
```

Comment:
You have "test 52" here and 53 in the path...

Actually, I already took 53 in #16428, so since you need to adjust yours, would you mind taking 54 instead?

Thanks!
Filipe

Reply:
Changed to 54, thanks